Add Spanish language support and translations

TeenBiscuits opened the pull request:
Add Spanish localization

TeenBiscuits commented:
Emmmm, what?

vsc55 commented:
Hi @TeenBiscuits,
It appears the system failed during the check.
Try creating a new pull request or a new commit to your repository in the same branch to see if the system runs the check again.

TeenBiscuits commented:
Added. It's not a copy of the other value. `column.http-code` translates to "Access" in English while `redirection-host.forward-http-code` is "HTTP Code". I think this is a mistake; I didn't notice it while translating to Spanish. I fixed it in my last commit
